utils/_variables.scss

```diff
@@ -1,188 +1,6 @@
-  "resting": $elevation1,
-  "raised": $elevation2,
-  "hovering": $elevation3,
-  "floating": $elevation5
-);
-$z-layers: (
-  "goku":            9001,
-  "shoryuken":       8000,
-  "upper":             10,
-  "default":            1,
-  "below":             -1,
-  "bottomless-pit": -9000
-);
-$base-font-color: $asphalt;
-$color-text-link: $liverleaf;
-$color-text-inverse: $white;
-$color-text-success: $kohpayam;
-$color-text-error: $flamingo;
-$color-text-button-secondary: $asphalt;
-$color-text-button-action: $liverleaf;
-$color-icon-table-filter: $miroschka;
-$color-icon-table-header-sort: $asphalt;
-$color-text-table-column-header: $asphalt;
-$color-icon-input: $french-gray;
-$action-color: $primary-color;
-$secondary-background-color: $catkin;
-$color-button-primary: $liverleaf;
-$color-background-table-row: $white;
-$color-background-table-row-alt: $catkin;
-$color-background-table-row-hover: $sail;
-$color-background-dropdown-option-hover: $sail;
-$color-background-accordion-segment: $miroschka;
-$color-background-table-filters: $catkin;
-$color-background-progress-bar: $miroschka;
-$base-background-color: $white;
-$shadow-shallow-elevation: map-get($elevation-shadows, "resting");
-$shadow-inset-shadow: inset 0 1px 1px 0 rgba($black, 0.22), inset 0 2px 4px 0 rgba($santas-gray, 0.65);
-$recessed-paper-shadow-primary: inset 0 1px 2px 0 rgba($black, 0.19);
-$recessed-paper-shadow-secondary: inset 0 1px 14px 0 rgba($black, 0.08);
-$form-box-shadow: inset 0 1px 3px rgba($black, 0.06);
-$form-box-shadow-focus: $form-box-shadow, 0 0 5px adjust-color($action-color, $lightness: -5%, $alpha: -0.3);
-$base-border-color: $black;
-$color-border-table-row-secondary: $catkin;
-$base-border: 1px solid $base-border-color;
-$color-border-table: 1px solid $base-border-color;
-$color-border-progress-bar: $french-gray;
-$color-border-input-default: $french-gray;
-$color-border-input-highlight: $liverleaf;
-$color-border-input-warning: $zalando-orange;
-$color-border-input-error: $black;
-$refills-serif: 'Lusitana', 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
-$refills-narrow-sans: 'Oswald', sans-serif;
-$refills-second-serif: 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
-$refills-nav-height: 60px;
-$refills-body-background-color: $white;
-$refills-color: $shark;
-$refills-accent-color: $liverleaf;
-$refills-logo-color: $black;
-$refills-anchor-color: $liverleaf;
-$refills-hero-background: $french-gray;
-$refills-secondary-anchor-color: $black;
-$refills-highlight-color: $liverleaf;
-$refills-white-shadow: 0 1px 1px transparentize($white, 0.6);
-$refills-medium-screen: em(640);
-$refills-large-screen: em(860);
-//Tooltip Settings
-
-//Input Variables
-$input-border-color: $color-border-input-default;
-$input-shadow-color: lighten($input-border-color, 15%);
-$input-shadow-inset: inset 0px 1px 3px 0px $input-shadow-color;
-
-$default-feature: min-width;
-$column: 90px;
-$gutter: 30px;
-$grid-columns: 12;
-$max-width: 1200px;
```

Comment:
Only used in the current ws-header.
Yes right, we should take out in the new header and just use the grid